cms/static/js/spec/views/settings/main_spec.js

```diff
@@ -182,5 +204,103 @@ define([
             AjaxHelpers.expectJsonRequest(requests, 'POST', urlRoot, modelData);
         });
 
+        it('can add learning information', function () {
+            this.view.$(SELECTORS.add_course_learning_info).click();
+            expect('click').not.toHaveBeenPreventedOn(SELECTORS.add_course_learning_info);
+            expect(this.model.get('learning_info').length).toEqual(2);
+            this.view.$(SELECTORS.add_course_learning_info).click();
+            expect(this.model.get('learning_info').length).toEqual(3);
+        });
+
+        it('can delete learning information', function () {
+            for (var i = 0 ; i < 2; i++) {
+                this.view.$(SELECTORS.add_course_learning_info).click();
+            }
+            expect(this.model.get('learning_info').length).toEqual(3);
+            expect(this.view.$(SELECTORS.delete_course_learning_info)).toExist();
+            this.view.$(SELECTORS.delete_course_learning_info).click();
+            expect(this.model.get('learning_info').length).toEqual(2);
+        });
+
+        it('can save learning information', function () {
+            expect(this.model.get('learning_info').length).toEqual(1);
+            var requests = AjaxHelpers.requests(this),
+                expectedJson = $.extend(true, {}, modelData, {
+                    learning_info: ['testing info']
+                });
+
+            // Input some value.
+            this.view.$("#course-learning-info-0").val('testing info');
+            this.view.$("#course-learning-info-0").trigger('change');
+            this.view.saveView();
+            AjaxHelpers.expectJsonRequest(
+                requests, 'POST', urlRoot, expectedJson
+            );
+            AjaxHelpers.respondWithJson(requests, expectedJson);
+
+        });
+
+        it('can add instructor information', function () {
+            this.view.$(SELECTORS.add_course_instructor_info).click();
+            expect(this.model.get('instructor_info').instructors.length).toEqual(2);
+            this.view.$(SELECTORS.add_course_instructor_info).click();
+            expect(this.model.get('instructor_info').instructors.length).toEqual(3);
+
+        });
+
+        it('can delete instructor information', function () {
+            this.view.$(SELECTORS.add_course_instructor_info).click();
+            expect(this.model.get('instructor_info').instructors.length).toEqual(2);
+            expect(this.view.$(SELECTORS.remove_instructor_data)).toExist();
+            this.view.$(SELECTORS.remove_instructor_data).click();
+            expect(this.model.get('instructor_info').instructors.length).toEqual(1);
```

Comment:
If you can't delete the instructor (which seems a bit odd to me), you should add a test that delete is not an option in that case.